whatToDad_site/whatToDad_app/views.py

```diff
@@ -33,7 +34,66 @@ def get(self, request):
         )
 
     def post(self, request):
+
         post_form = PostForm(request.POST)
         post_form.save()
 
-        return redirect('home')
\ No newline at end of file
+        return redirect('home')
+
+class ActivityPage(View):
+    def get(self, request):
+
+        activities = Activity.objects.all()
+        activity_form = ActivityForm()
+
+        html_data = {
+            'activities': activities,
+            'form': activity_form,
+        }
+
+        return render(
+            request=request,
+            template_name='activity.html',
+            context= html_data
+        )
+
+    def post(self, request):
+
+        if 'add' in request.POST: 
```

Comment:
What is this checking? Won't the `add` field be in the body based on the HTML, even if it's an empty string?